README.md

```diff
@@ -292,6 +314,28 @@ In build/platforms/android/project.properties, replace
 
 and perform a Gradle sync.
 
+### Failed to load resource: net::ERR_CLEARTEXT_NOT_PERMITTED
+
+Starting with Android 9 (API level 28), cleartext support is disabled by default. In order to fix this:
+
+In `src/config.xml.mustache` add this *attribute* `xmlns:android="http://schemas.android.com/apk/res/android"` to your widget tag at the beginning of the file. So it must be look like
+
+```
+<widget xmlns = "http://www.w3.org/ns/widgets"
+        xmlns:gap = "http://phonegap.com/ns/1.0"
+        id        = "{{identifier}}"
+        version   = "{{version}}" 
+        xmlns:android="http://schemas.android.com/apk/res/android">

```

Comment:
This missing tag is not specific to clearText. It indeed needs to be fixed in the mendix-hybrid-app-base. Feel free to open a PR for it; non the less I will add it to the our backlog.

Reply:
Created PR: `https://github.com/mendix/hybrid-app-base/pull/43`. Already tested against this branch

Reply:
Thx for Merging the PR :)